src/main/resources/assets/tconstruct/lang/zh_cn.json

```
@@ -806,11 +878,62 @@
   "fluid.tconstruct.molten_refined_glowstone": "熔融强化荧石",
   "fluid.tconstruct.flowing_molten_refined_glowstone": "流动的熔融强化荧石",
   "item.tconstruct.molten_refined_glowstone_bucket": "熔融强化荧石桶",
+  "fluid.tconstruct.molten_refined_glowstone.fluid_effect": "熔融强化荧石十分适合追踪目标，还不会牺牲很多伤害。",
 
   "block.tconstruct.molten_refined_obsidian_fluid": "熔融强化黑曜石",
   "fluid.tconstruct.molten_refined_obsidian": "熔融强化黑曜石",
   "fluid.tconstruct.flowing_molten_refined_obsidian": "流动的熔融强化黑曜石",
   "item.tconstruct.molten_refined_obsidian_bucket": "熔融强化黑曜石桶",
+  "fluid.tconstruct.molten_refined_obsidian.fluid_effect": "熔融强化黑曜石打击装甲单位可是一把好手。",
+
+
+  "_comment": "Fluid effect contents",
+  "fluid_effect.tconstruct.separator": "%s；%s",
+
+  "fluid_effect.tconstruct.extinguish_fire": "灭火",
+  "fluid_effect.tconstruct.teleport": "传送目标",
+  "fluid_effect.tconstruct.calcified": "+10秒钙质化",
+
+  "fluid_effect.tconstruct.cure_effects": "解除大多数状态效果",
+  "fluid_effect.tconstruct.remove_effect": "移除%s",
+  "fluid_effect.tconstruct.mob_effect.add": "+%d秒%s",
+  "fluid_effect.tconstruct.mob_effect.set": "%d秒%s",
+  "fluid_effect.tconstruct.potion": "添加指定药水效果",
+
+  "fluid_effect.tconstruct.restore_hunger": "恢复%d饥饿值",
+  "fluid_effect.tconstruct.fire.add": "+%d秒着火时间",
+  "fluid_effect.tconstruct.fire.set": "%d秒着火时间",
+  "fluid_effect.tconstruct.freeze.add": "+%d秒冰冻时间",
+  "fluid_effect.tconstruct.freeze.set": "%d秒冰冻时间",
+  "fluid_effect.tconstruct.award_stat.add": "+%s%s",
+  "fluid_effect.tconstruct.award_stat.subtract": "-%s%s",
+  "fluid_effect.tconstruct.add_breath.add": "+%d氧气值",
+  "fluid_effect.tconstruct.add_breath.subtract": "-%d氧气值",
+  "fluid_effect.tconstruct.push_entity.push": "推动目标",
+  "fluid_effect.tconstruct.push_entity.pull": "拉动目标",
+
+  "fluid_effect.tconstruct.damage": "%s伤害",
+  "fluid_effect.tconstruct.damage.tconstruct.fluid.impact": "%s伤害",
+  "fluid_effect.tconstruct.damage.tconstruct.fluid.fire": "%s火焰伤害",
+  "fluid_effect.tconstruct.damage.tconstruct.fluid.cold": "%s寒冷伤害",
+  "fluid_effect.tconstruct.damage.tconstruct.fluid.magic": "%s魔法伤害",
+  "fluid_effect.tconstruct.damage.tconstruct.fluid.water": "%s溺水伤害",
+  "fluid_effect.tconstruct.damage.tconstruct.fluid.spike": "%s穿刺伤害",
+
+  "fluid_effect.tconstruct.place_block": "放置%s",
+  "fluid_effect.tconstruct.place_block.held": "放置手持方块",
+  "fluid_effect.tconstruct.break_block": "采掘方块",
+  "fluid_effect.tconstruct.break_block.separator": "%s，%s",
+  "fluid_effect.tconstruct.break_block.enchanted": "%s采掘方块",
+  "fluid_effect.tconstruct.break_block.hardness": "采掘硬度最高为%s的方块",
+  "fluid_effect.tconstruct.break_block.enchanted.hardness": "%2$s采掘硬度最高为%1$s的方块",
```

```suggestion
  "fluid_effect.tconstruct.break_block.enchanted.hardness": "附带%2$s采掘硬度最高为%1$s的方块",
```
The second parameter here should be enchantment names, it's better to add smth close to "with" in front of it.